packages/amplify-category-auth/src/provider-utils/awscloudformation/utils/check-for-auth-migration.ts

```diff
@@ -10,10 +11,15 @@ export const checkAuthResourceMigration = async (context: $TSContext, authName:
   if (!imported) {
     const cliState = new AuthInputState(authName);
     if (!cliState.cliInputFileExists()) {
-      printer.debug('Cli-inputs.json doesnt exist');
+      printer.debug("cli-inputs.json doesn't exist");
       // put spinner here
       const headlessMigrate = context.input.options?.yes || context.input.options?.forcePush || context.input.options?.headless;
-      if (headlessMigrate || (await prompter.yesOrNo(`Do you want to migrate this ${authName} to support overrides?`, true))) {
+      const docsLink = 'https://docs.amplify.aws/cli/migration/overrides';
+      const migrateResourceMessage =
+        `Do you want to migrate ${AmplifyCategories.AUTH} resource "${authName}" to support overrides?` +
+        `${EOL}(Recommended to try in a non-production environment first: "amplify env add".) Learn more about this migration: ${docsLink}`;
```

Comment:
Can we put this in line in red - "Recommended to try in a non-production environment first. To create/clone a new environment run "amplify env add".

@renebrandel thoughts?

Reply:
Updated